Makefile

```diff
@@ -148,7 +148,7 @@ dbuild: crioimage
 	docker run --name=${CRIO_INSTANCE} -e BUILDTAGS --privileged -v ${PWD}:/go/src/${PROJECT} --rm ${CRIO_IMAGE} make binaries
 
 integration: crioimage
-	docker run -e STORAGE_OPTIONS="--storage-driver=vfs" -e TEST_USERNS -e TESTFLAGS -e TRAVIS -t --privileged --rm -v ${CURDIR}:/go/src/${PROJECT} ${CRIO_IMAGE} make localintegration
+	docker run -e STORAGE_OPTIONS="--storage-driver=vfs" -e TEST_USERNS -e TESTFLAGS -e TRAVIS -t --privileged --rm -v ${CURDIR}:/go/src/${PROJECT} ${CRIO_IMAGE} make localintegration BUILDTAGS='btrfs_noversion libdm_no_deferred_remove seccomp selinux containers_image_ostree_stub libpod'
```

Comment:
Instead of hardcoding these, you could use:

```
... make BUILDTAGS='$(BUILDTAGS)' localintegration
```

to pass the variable through to the container.

Reply:
ok fixed